website-v2/src/routes/+page.svelte

```diff
@@ -1,133 +1,176 @@
-                        calculs permet de vous conformer à l’obligation de transparence des algorithmes
-                        publics.
-                    </li>
-                </ol>
-            </Card>
-        </div>
-    </section>
-    <section class=" mt-12">
-        <h2>Du code clair et lisible.</h2>
-        <p>
-            Essayez de modifiez le <strong>salaire brut</strong> à
-            <strong>3000 €/mois</strong> dans l'exemple suivant...
+<main class="">
+    <section class="mt-36 flex flex-col items-center justify-center gap-8">
+        <p class="max-w-5xl text-center text-2xl font-normal">
+            Publicodes permet de modéliser des <span class="font-regular"
+                >domaines métiers complexes</span
+            >, en les décomposant en <span class="font-regular">règles élémentaires simples</span>
+            qui soient
+            <span class="font-regular">lisibles par tout le monde</span>.
         </p>
-        <div class="max-sm:-mx-6">
-            <PublicodesEditor
-                title="Calcul du salaire net"
-                showDocByDefault
-                hideDocButton
-                size="LG"
-                code={`salaire brut: 2500 €/mois
+        <img
+            src={PublicodesSchemaSVG}
+            alt="Schéma de fonctionnement de
+		Publicodes"
+            class="mt-12 w-full max-w-4xl"
+        />
+    </section>
+    <section class="mt-36 flex flex-col gap-8">
+        {@render banner(microscope, 'bg-publicodes-orange bg-opacity-10', 'Pour les expert·es', [
+            {
+                subtitle: 'Low code',
+                content:
+                    'Entre le no-code et le code, Publicodes est compréhensible par toute personne ayant déjà manipulé un tableau Excel.'
+            },
+            {
+                subtitle: 'Transparent',
+                content:
+                    "Plus d'erreurs cachées : pour vérifier qu’une règle est correctement implémentées, il suffit de la lire."
+            },
+            {
+                subtitle: 'Un langage commun',
+                content:
+                    'Développeurs, expert·es : tout le monde parle la même langue. Cela veut dire une communication plus fluide, et moins d’erreurs.'
+            }
+        ])}
+        {@render banner(
+            microscope,
+            'bg-publicodes-green bg-opacity-10',
+            'Pour les développeur·euses',
+            [
+                {
+                    subtitle: 'Clean architecture',
+                    content:
+                        'Publicodes favorise la maintenabilité et la testabilité en découplant clairement le code métier du code applicatif.'
+                },
+                {
+                    subtitle: 'Facile à prendre en main',
+                    content:
+                        "Il dispose d'une extension VSCode dédiée (coloration syntaxique, goto def, auto-complétion…) et est basé sur la syntaxe YAML."
```

Comment:
Un détail mais généralement j'évite de mettre du contenu html dans une chaîne de caractère JS car on pert toute la flexibilité du `.svelte`, pour utiliser du html dans le texte (il y avait ici un lien `<a>` vers l'extension VSCode) ou composant custom.

Reply:
Oui je me le suis noté. Ceci dit, je ne trouve pas forcément logique d'avoir juste un seul lien pour l'extension ici.

Reply:
Tout à fait d'accord. Il faudrait que les arguments de la snippet soient des snippet elles même (en tout cas pour la description). Et si pas possible, il ne faut pas hésiter à créer un composant svelte, ce sera plus clair !